zmk/templates/board/nrf52840/common_outer.dtsi

```diff
@@ -14,46 +14,50 @@
     status = "okay";
 };
 
-&usbd {
-    status = "okay";
+&uart0 {
+    compatible = "nordic,nrf-uarte";
+    current-speed = <115200>;
+    pinctrl-0 = <&uart0_default>;
+    pinctrl-1 = <&uart0_sleep>;
+    pinctrl-names = "default", "sleep";
+};
 
-    cdc_acm_uart: cdc_acm_uart {
-        compatible = "zephyr,cdc-acm-uart";
-    };
+zephyr_udc0: &usbd {
+    status = "okay";
 };
 
 &flash0 {
     // Adjust this flash map as is necessary for your board. For more information, see
-    // https://docs.zephyrproject.org/latest/reference/storage/flash_map/flash_map.html
+    // https://docs.zephyrproject.org/3.6.0/services/storage/flash_map/flash_map.html#relationship-with-devicetree
```

Comment:
```suggestion
    // https://docs.zephyrproject.org/3.5.0/services/storage/flash_map/flash_map.html#relationship-with-devicetree
```
Seems like the same page, so linking for ZMK's version might be better.

Reply:
Looks like I had accidentally used 3.6.0 for all the other links, so I just updated this to match. Switched everything back to 3.5.0.